cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/discover.spec.js

```diff
@@ -26,6 +26,17 @@ const indexSet = [
 
 describe('discover app', { scrollBehavior: false }, () => {
   before(() => {
+    if (Cypress.env('SECURITY_ENABLED')) {
+      /**
+       * Security plugin is using private tenant as default.
```

Comment:
Would mind to leave a comment of why changing default tenant to private make the tests to fail?

Reply:
![image](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/13493605/116c9061-70ff-4ff0-93b5-e29f92d2e7d2)
It seems the saved search page is unable to load as it is trying to fetch a index pattern in its default tenant(private) however it is inside a global tenant when doing search. Still need a deeper dive on the source code of these two pages.